### DISCUSSION TITLE
export to YNAB

### DISCUSSION THREAD

**Discussion by slln** [actualbudget/releases#254] [Q&A]
Hate that I have to ask and do this, but is there an option to export/ import to YNAB?

**Comment by Mreco99** [comment]
Dont do it!!!

**Reply by slln** [reply]
Any other option than wasting so much time hosting my server? Not to even mention the mobile apps...

**Reply by cuken** [reply]
As far as I can tell, the export from actual dumps a metadata.json and a sqlite file. I don't have a paid actual account so I'm not sure if this is the same functionality in the hosted version. Can you confirm this is what you see?

**Reply by slln** [reply]
Indeed, same on paid version sync via iCloud, there is folder with the json and sqlite.

**Reply by cuken** [reply]
Looking over the YNAB docs, it looks like this might be slightly painful to do. The best route forward I think would be to parse the SQLite file, split it into a `.csv` file per account and import them via YNABs csv import process.  I should be able to hack something together pretty quickly to do this.

**Reply by cuken** [reply]
Wasn't as bad as I thought it would be, curious if this will work; Code for anyone interested is here:
https://github.com/cuken/ActualExporter

**Reply by slln** [reply]
Wow, great work!
Please excuse my ignorance but can you point to some info on how to use the utility?

**Reply by cuken** [reply]
The instructions are in the repo under the Usage section.

**Reply by Kaboom6253848** [reply]
Hi there - I ran into the following error message. Any idea how to tackle this?

`Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub`

**Reply by cuken** [reply]
@Kaboom6253848 I replied on the source repo, let me know what you get. Thanks!

**Reply by patrikarvidsson** [reply]
In case you're not hell-bent on going over to YNAB (and the mobile-app factor isn't a big deal) you could host Actual with PikaPods for ~$1.4 per month:

https://www.pikapods.com/apps

Edit: I'm not affiliated with them, but I do use both PikaPods and BorgBackup and love both services.
